widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/crystal/globalCrystal.css

```diff
@@ -122,6 +122,12 @@ SELECT,
   padding: 0 0 0 4px;
   font-size: 13px;
 }
+
+select option {
+  width: fit-content;
+  min-width: 100%;
+}
```

Comment:
Hello @phalguni-a 

I was testing some of Analyzer's dialogs and when this css was included, it introduced scroll bars at least in one of them. Taking that into account as well the fact that the selector is super generic, I feel there is a big probability of unwanted consequences in other places.

Couldn't this be combined with a Browse Files perspective class so that this change only affects this perspective?

Reply:
Replying to the comment on Teams:

The was on a select with `size != 1` that also has `padding` styles. So the scrollbar results from the `min-width 100%` + the padding.

Other way to solve this would be to include a `padding: 0` in this selector, but could also have unwanted consequences, so I'm not sure if it would be the best approach

Reply:
@davidmsantos90 
I agree the rule is very general. However, it also seems harmless and appears to solve a styling issue that all listboxes may suffer from. It definitely should be targeting listboxes only, though.

Would like to understand exactly what unwanted consequences it is causing.

Are vertical scrollbars being displayed in listboxes, and weren't before?
Or, is it causing the dialog's vertical scrollbar to be displayed sooner?
Is it horizontal scrollbars of listboxes?
Could it be an issue with the `box-sizing` used for the `option` elements, causing `100%` to overflow the parent?

Reply:
Not vertical, just horizontal and even when the dialog is at "full" size

![Screenshot 2023-03-08 at 19 41 28](https://user-images.githubusercontent.com/6906072/223827264-a6e8e938-fcb9-40f9-9d13-7319a56d0bae.png)

By default css doesn't discount the element padding when using `*width: 100%`, causing the overflow
To keep the padding style in conjunction with `min-width: 100%`, we can also add `box-sizing: border-box`

Reply:
Concluding, such a rule would require knowledge of the elements' `box-sizing` and/or `paddings`.  Because that is not known in general, nor can be changed in general, more specific rules must be used instead.

Thanks, @davidmsantos90!

Reply:
Sure, I agree we can have a specific rule and we can certainly add other selectors from other list boxes to the same rule if we come across same scenario.

Thanks @davidmsantos90 and @dcleao .